### ISSUE TITLE
Says that .dll files are missing when they are in the folder visible

### ISSUE THREAD

**Issue by BananaManaa** [Drewol/unnamed-sdvx-clone#540]
Says that "archive.dll", "SDL2.dll", "vorbisfile.dll", and "jpeg62.dll" is missing when it is visibly in the folder.

Happens as soon as you launch the game

 - OS: Windows 10
 - GPU: Intel Hd
 - CPU: i3-3240
 - Version: 0.5.0

**Comment by Hinara** [comment]
Are the dll in the same folder as you launch the game ?

**Comment by BananaManaa** [comment]
Yes, its all in the same folder. I even tried uninstalling and reinstalling the game and tried adding the .dlls on their own (downloading them then adding them into the folder)
